collector/collector_test.go

```diff
@@ -576,7 +569,8 @@ func TestGetDynamicNamespaceElements(t *testing.T) {
 				Namespace: []configReader.Namespace{
 					configReader.Namespace{Source: "string", String: "test1"},
 					configReader.Namespace{Source: "string", String: "test2"},
-					configReader.Namespace{Source: "snmp", Oid: ".1.3.6.1.2.1.1.9.1.3", Name: "name"},
+					configReader.Namespace{Source: "snmp", Oid: ".1.3.6.1.2.1.1.9.1.3", Name: "name1"},
+					configReader.Namespace{Source: "snmp", Oid: ".1.3.6.1.2.1.1.9.1.7", Name: "name2"},
```

Comment:
This test case is to test `getDynamicNamespaceElements` ```with incorrect configuration of dynamic elements of namespace, the number of results is not equal the number of dynamic elements of namespace``` in this pull request you are removing this condition, so this test case should be removed/replaced with different one.